parts/k8s/cloud-init/artifacts/cis.sh

```diff
@@ -8,7 +8,7 @@ assignRootPW() {
     CMD="import crypt, getpass, pwd; print crypt.crypt('$SECRET', '\$6\$$SALT\$')"
     HASH=`python -c "$CMD"`
 
-    echo 'root:'$HASH | /usr/sbin/chpasswd -e 2>/dev/null;
+    echo 'root:'$HASH | /usr/sbin/chpasswd -e || exit $ERR_CIS_ASSIGN_FILE_PERMISSION
```

Comment:
@jackfrancis thoughts on this? Any reason why we actually want to throw away the stderr?

Reply:
no, this was just copy pasta from the original work item